scripts/demo/sv4d_helpers.py

```diff
@@ -761,14 +886,13 @@ def denoiser(input, sigma, c):
                     return model.denoiser(
                         model.model, input, sigma, c, **additional_model_inputs
                     )
-
                 load_module_gpu(model.model)
                 load_module_gpu(model.denoiser)
                 samples_z = sampler(denoiser, randn, cond=c, uc=uc)
                 unload_module_gpu(model.model)
                 unload_module_gpu(model.denoiser)
-
                 load_module_gpu(model.first_stage_model)
+                model.en_and_decode_n_samples_a_time = decoding_t
```

Comment:
Is "en_and_decode_n_samples_a_time" the same as "decoding_t"? If so, should we merge the two parameters?

Reply:
encoding_t and decoding_t are slightly different in memory consumption so one number for these two might not be reasonable. I separate this two parameters and the users can adjust both of them.